src/components/calcite-pick-list-item/calcite-pick-list-item.tsx

```diff
@@ -29,7 +29,7 @@ export class CalcitePickListItem {
   // --------------------------------------------------------------------------
 
   /**
-   * Compact removes the selection icon (radio or checkbox) and adds a compact attribute.
+   * ** DEPRECATED ** Compact removes the selection icon (radio or checkbox) and adds a compact attribute.
    * This allows for a more compact version of the `calcite-pick-list-item`.
```

Comment:
Needs the `@deprecated` here instead of `**`

Reply:
See the other deprecated examples.

Reply:
Applies to other uses of deprecated below.

Reply:
https://github.com/Esri/calcite-app-components/blob/14361950c8b116a04c7609058fac1045842c97ea/src/components/calcite-block-section/calcite-block-section.tsx#L50

Reply:
oh glob! ok will fix